Docker ERROR: failed to solve: process "/bin/sh -c make build" did not complete successfully: exit code: 2

Hello,
during the build of the docker I receive error during the stage 16 (RUN make build).
The command I executed:
`sudo docker build -t lightpanda:latest .`

The full log:
```
[+] Building 220.8s (19/21)                                                                              docker:default
 => [internal] load build definition from Dockerfile                                                               0.0s
 => => transferring dockerfile: 2.34kB                                                                             0.0s
 => [internal] load metadata for docker.io/library/ubuntu:22.04                                                    1.4s
 => [internal] load .dockerignore                                                                                  0.0s
 => => transferring context: 2B                                                                                    0.0s
 => [stage-0  1/16] FROM docker.io/library/ubuntu:22.04@sha256:0e5e4a57c2499249aafc3b40fcd541e9a456aab7296681a399  0.0s
 => => resolve docker.io/library/ubuntu:22.04@sha256:0e5e4a57c2499249aafc3b40fcd541e9a456aab7296681a3994d63158720  0.0s
 => => sha256:3d1556a8a18cf5307b121e0a98e93f1ddf1f3f8e092f1fddfd941254785b95d7 424B / 424B                         0.0s
 => => sha256:97271d29cb7956f0908cfb1449610a2cd9cb46b004ac8af25f0255663eb364ba 2.30kB / 2.30kB                     0.0s
 => => sha256:0e5e4a57c2499249aafc3b40fcd541e9a456aab7296681a3994d631587203f97 6.69kB / 6.69kB                     0.0s
 => [stage-0  2/16] RUN apt-get update -yq &&     apt-get install -yq xz-utils         python3 ca-certificates g  35.5s
 => [stage-0  3/16] RUN curl -L -O https://github.com/jedisct1/minisign/releases/download/0.11/minisign-0.11-linu  1.1s
 => [stage-0  4/16] RUN curl -O https://ziglang.org/download/0.13.0/zig-linux-x86_64-0.13.0.tar.xz &&     curl -O  1.7s
 => [stage-0  5/16] RUN minisign-linux/x86_64/minisign -Vm zig-linux-x86_64-0.13.0.tar.xz -P RWSGOq2NVecA2UPNdBUZ  0.3s
 => [stage-0  6/16] RUN rm -fr minisign-0.11-linux.tar.gz minisign-linux                                           0.2s
 => [stage-0  7/16] RUN tar xvf zig-linux-x86_64-0.13.0.tar.xz &&     mv zig-linux-x86_64-0.13.0 /usr/local/lib &  3.4s
 => [stage-0  8/16] RUN rm -fr zig-linux-x86_64-0.13.0.tar.xz zig-linux-x86_64-0.13.0.tar.xz.minisig               0.1s
 => [stage-0  9/16] RUN cat <<EOF > /root/.gitconfig                                                               0.2s
 => [stage-0 10/16] RUN git clone git@github.com:lightpanda-io/browser.git                                         1.3s
 => [stage-0 11/16] WORKDIR /browser                                                                               0.0s
 => [stage-0 12/16] RUN git submodule init &&     git submodule update --recursive                                11.2s
 => [stage-0 13/16] RUN cd vendor/zig-js-runtime &&     git submodule init &&     git submodule update --recursiv  2.6s
 => [stage-0 14/16] RUN make install-libiconv &&     make install-netsurf &&     make install-mimalloc            65.0s
 => [stage-0 15/16] RUN curl -L -o libc_v8.a https://github.com/lightpanda-io/zig-v8-fork/releases/download/v0.1.  2.5s
 => ERROR [stage-0 16/16] RUN make build                                                                          94.1s
------
 > [stage-0 16/16] RUN make build:
0.198 Building (release safe)...
94.01 install
94.01 +- install lightpanda
94.01    +- zig build-exe lightpanda ReleaseSafe native 2 errors
94.01 error: ld.lld: undefined symbol: v8__Isolate__SetData
94.01     note: referenced by v8.zig:418 (/browser/vendor/zig-js-runtime/vendor/zig-v8/src/v8.zig:418)
94.01     note:               /browser/.zig-cache/o/05af3308be9be989b0dc99fab2874e4f/lightpanda.o:(engines.v8.v8.Env.start)
94.01 error: ld.lld: undefined symbol: v8__Isolate__GetData
94.01     note: referenced by v8.zig:422 (/browser/vendor/zig-js-runtime/vendor/zig-v8/src/v8.zig:422)
94.01     note:               /browser/.zig-cache/o/05af3308be9be989b0dc99fab2874e4f/lightpanda.o:(engines.v8.v8.Env.resolveModuleCallback)
94.01 error: the following command failed with 2 compilation errors:
94.01 /usr/local/lib/zig-linux-x86_64-0.13.0/zig build-exe -OReleaseSafe --dep jsruntime --dep netsurf --dep asyncio --dep tls --dep websocket -Mroot=/browser/src/main.zig /browser/vendor/zig-js-runtime/vendor/v8/x86_64-linux/release/libc_v8.a -OReleaseSafe --dep jsruntime_build_options --dep tigerbeetle-io --dep v8 -Mjsruntime=/browser/vendor/zig-js-runtime/src/api.zig /browser/vendor/libiconv/out/linux-x86_64/lib/libiconv.a /browser/vendor/netsurf/out/linux-x86_64/lib/libdom.a /browser/vendor/netsurf/out/linux-x86_64/lib/libhubbub.a /browser/vendor/netsurf/out/linux-x86_64/lib/libparserutils.a /browser/vendor/netsurf/out/linux-x86_64/lib/libwapcaplet.a -I /browser/vendor/libiconv/out/linux-x86_64/lib/libiconv.a -I /browser/vendor/netsurf/out/linux-x86_64/include -I /browser/vendor/netsurf/libdom/src -I /browser/vendor/netsurf/libhubbub/src -I /browser/vendor/netsurf/libparserutils/src -I /browser/vendor/netsurf/libwapcaplet/src --dep mimalloc --dep jsruntime -Mnetsurf=/browser/src/netsurf/netsurf.zig -Masyncio=/browser/vendor/zig-async-io/src/lib.zig -Mtls=/browser/vendor/tls.zig/src/main.zig -Mwebsocket=/browser/vendor/websocket.zig/src/websocket.zig -Mjsruntime_build_options=/browser/.zig-cache/c/6cb9efbd9acc368d5c5d2e1c164feb71/options.zig -Mtigerbeetle-io=/browser/vendor/zig-js-runtime/vendor/tigerbeetle-io/io.zig -I /browser/vendor/zig-js-runtime/vendor/zig-v8/src -Mv8=/browser/vendor/zig-js-runtime/vendor/zig-v8/src/v8.zig /browser/vendor/mimalloc/out/linux-x86_64/lib/libmimalloc.a -I /browser/vendor/mimalloc/include -Mmimalloc=/browser/src/mimalloc/mimalloc.zig -lc++ --cache-dir /browser/.zig-cache --global-cache-dir /root/.cache/zig --name lightpanda --listen=-
94.01 Build Summary: 1/4 steps succeeded; 1 failed (disable with --summary none)
94.01 install transitive failure
94.01 +- install lightpanda transitive failure
94.01    +- zig build-exe lightpanda ReleaseSafe native 2 errors
94.01 error: the following build command failed with exit code 1:
94.01 /browser/.zig-cache/o/6a578bece6eebf9e4d163b2088f2ff3f/build /usr/local/lib/zig-linux-x86_64-0.13.0/zig /browser /browser/.zig-cache /root/.cache/zig --seed 0x36588392 -Ze071bbb86038cc8c -Doptimize=ReleaseSafe -Dengine=v8
94.02 Build ERROR
94.02 make: *** [Makefile:60: build] Error 1
------
Dockerfile:68
--------------------
  66 |
  67 |     # build release
  68 | >>> RUN make build
  69 |
  70 |     FROM ubuntu:22.04
--------------------
ERROR: failed to solve: process "/bin/sh -c make build" did not complete successfully: exit code: **2**
```

--- a/Dockerfile
+++ b/Dockerfile
@@ -5,7 +5,7 @@ ARG ZIG_MINISIG=RWSGOq2NVecA2UPNdBUZykf1CCb147pkmdtYxgb3Ti+JO/wCYvhbAb/U
 ARG OS=linux
 ARG ARCH=x86_64
 ARG V8=11.1.134
-ARG ZIG_V8=v0.1.9
+ARG ZIG_V8=v0.1.11
 
 RUN apt-get update -yq && \
     apt-get install -yq xz-utils \

